page.php

```diff
@@ -26,6 +26,7 @@
 $context['post'] = $post;
 
 $page_meta_data = get_post_meta( $post->ID );
+
 $context['header_title']        = $page_meta_data['p4_title'][0] === null ? $post->title : $page_meta_data['p4_title'][0];
```

Comment:
To follow WP coding standards use [Yoda conditions](https://make.wordpress.org/core/handbook/best-practices/coding-standards/php/#yoda-conditions) and place the value on the left side of the operator like
null === $page_meta_data['p4_title'][0]